js/index.js

```diff
@@ -1,7 +1,46 @@
-// Iteration 1: Names and Input
+let hacker1 = "Gosia";
+let hacker2 = "Hugo";
 
+for (let i=0; i < hacker2.length; i++) {
+  console.log(hacker2[i].toUpperCase());
+}
 
-// Iteration 2: Conditionals
+for (let i = hacker2.length-1; i >= 0; i--) {
+  console.log(hacker2[i]);
+}
 
+if (hacker1[0] < hacker2[0]) {
+  console.log('The drivers name goes first');
+} else if (hacker2[0] < hacker1[0]) {
+  console.log('Yo, the navigator goes first definitely.')
+} else {
+  console.log('What?! You both have the same name?')
+};
```

Comment:
For this iteration, the goal is to compare both name's length! by checking if hacker1[0] < hacker2[0] you're comparing the two characters at index 0 in both names! Maybe you could make use of the length property? Ex: hacker.length